api/web3server_utils.go

```diff
@@ -386,12 +369,11 @@ func (svr *Web3Server) getLogsWithFilter(from uint64, to uint64, addrs []string,
 			return nil, err
 		}
 		ret = append(ret, logsObject{
-			BlockHash:       byteToHex(l.BlkHash),
-			TransactionHash: byteToHex(l.ActHash),
-			LogIndex:        uint64ToHex(uint64(l.Index)),
-			BlockNumber:     uint64ToHex(l.BlkHeight),
-			// TransactionIndex bug will be fixed in the next
-			TransactionIndex: "0x1",
```

Comment:
change 1 to `l.TxIndex` should be fine, could you verify with Wangxiong that this is safe? (there is NO particular App/client  expecting/using `TransactionIndex = 1` here